tests/autoyast/installation.pm

```diff
@@ -129,7 +129,7 @@ sub run {
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
     # So push a needle to check upcoming reboot on zVM that is a way to indicate the stage done
     # For bsc#1231522, need match reboot popup on powerVM to reboot after installation.
-    push @needles, 'autoyast-stage1-reboot-upcoming' if is_s390x || (is_pvm && !is_upgrade && is_sle('<15-SP7')) || (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'hvm'));
```

Comment:
Please add comments for the reason to exclude of hvm.

Reply:
Added comment there. please take a look again. thanks.

Reply:
This bug fix hasn't been backported yet, so should reserve the condition for <15-SP7.

Reply:
added and watit the VR to be run. thanks

Reply:
-[New VR](https://openqa.suse.de/tests/16313875#) passed, please take a look again. thanks!